nengo_spa/actions.py

```diff
@@ -26,10 +30,15 @@ class Parser(object):
         'reinterpret': Reinterpret,
         'translate': Translate}
 
-    def __init__(self, vocabs=None):
-        if vocabs is None:
-            vocabs = {}
-        self.vocabs = vocabs
+    def __init__(self, locals_=None):
+        if locals_ is None:
```

Comment:
I guess this could use some sort of comment because `locals_` can be one of three things:
* a dictionary,
* `None` in which case the locals of the calling function will be used (what is what a user calling that function usually wants),
* or an integer that defines of how many frames up the locals should be grabbed (this is used because other user-facing functions call this function and it it is easier to just increase this number by one than duplicating code to grabbing the locals dictionary of the calling function)